web/src/rest.js

```diff
@@ -5,9 +5,61 @@ import { RestClient } from '@girder/components/src';
 const apiRoot = process.env.VUE_APP_API_ROOT;
 const publishApiRoot = process.env.VUE_APP_PUBLISH_API_ROOT;
 
+// TODO remove the girderId if we can eliminate the girder ID from the URL
```

Comment:
Our frontend is still very girder-centric, so sometimes there is no choice but to accommodate that. This method translates the django schema from the publish API into something that can be easily rendered by the current frontend.
The girder object ID is not provided from the publish API, and it is required as both a unique key for rendering the list and to generate the route to navigate to when clicking on one of the dandisets. I pass it in to this helper, but I would much rather not have to do that. That's a problem for later though.

Reply:
I agree, and think it'd be better if we didn't rely on the girder ID in the route or here. However, that would necessitate a URL change, which may conflict with the redirector. I agree we should deal with this in a later issue.